app/src/createExcelFiles/ExcelUtils.js

```diff
@@ -0,0 +1,21 @@
+import * as XLSX from "xlsx";
+
+export function toExcelArrayOfArray(searchItems) {
+  const results = [["Title", "Links"]];
```

Comment:
You should use map here to change the data. it would look something like this.

Map automatically creates a new array for you. So nothing happens to searchItems array of object
```javascript
const result = searchItems.map((searchItem) => ([searchItem.title, searchItem.link]);
result.unshift(["Title", "Links"]);